CONTRIBUTING.rst

```diff
@@ -42,22 +42,28 @@ To update the pypy snapshot (and thus our python dependencies), run
 tahoe-lafs
 ..........
 
-We depend on pinned commit of tahoe-lafs.
-To update to the latest commit, run
+We test against a pinned commit of Tahoe-LAFS master.
+To update to the current master@HEAD revision, run:
 
 .. code:: shell
 
-   nix-shell --run 'niv update tahoe-lafs --branch master'
+   nix-shell --run 'niv update tahoe-lafs-master --branch master'
 
 It is also possible to pass ``pull/<pr-number>/head`` to test against a specific PR.
+When feasible we also test against a released version of Tahoe-LAFS.
+To update to a new release, run:
+
+.. code:: shell
+
+   nix-shell --run 'niv update --rev tahoe-lafs-A.B.C tahoe-lafs'
```

Comment:
This isn't entirely accurate, as this isn't *just* a version we test against, but also the version we run on storage nodes.